app/models/grda_warehouse/tasks/cleanup_client_search_queries_task.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module GrdaWarehouse::Tasks
+  class CleanupClientSearchQueriesTask
```

Comment:
Might be worth adding a note in [here](https://docs.google.com/document/d/12jh12ktye4XLJ1QxUYFpn23P9NoJVzMsQhkDOGwQiOQ/edit?tab=t.0#heading=h.ezuq9v3w5nts) that these are held for 2 years